pinecone/core/client/api_client.py

```diff
@@ -43,6 +43,12 @@
     validate_and_convert_types
 )
 
+try:
```

Comment:
I don't want to include numpy as a dependency only because I need to do a type check